integrations/database/postgresql/kubearchive.sql

```diff
@@ -88,6 +88,7 @@ CREATE TABLE public.log_url (
     uuid uuid NOT NULL,
     url text NOT NULL,
     container_name text NOT NULL,
+	json_path text NOT NULL,
```

Comment:
This should not be NOT NULL. It's possible that some logging system will not need it.

@maruiz93 , take note.

Reply:
done